### ISSUE TITLE
[Suggestion] Default additional/fuzzy items to new line

### ISSUE THREAD

**Issue by TekkadanPlays** [dillydill123/inventory-setups#291]
So far what I'm finding "clunky" about this new update is that fuzzy items seem to engulf the setup layout. The benefit here is that we can organize them freely. However most of my setups were created prior to the update. For setups without 'fuzzy' options, they seem great. For setups with fuzzy items, such as 3, 2, and 1-dose potions, they become mixed up in the empty spaces that exist beneficially.

Curious if additional items can default to a new line, below the setup. This was functioning on a few of my setups- I corrected at least one setup simply by toggling fuzzy options on all of the potions that already had fuzzy enabled. For one setup this completely fixed it yesterday. However for most other setups, it seems that fuzzy potions are still stacking up around the gear in the empty slots.

Overall this update is a game-changer, but I think additional items and fuzzy items should automatically default to a new line beneath the setup, where they can be later adjusted to suit our individual needs.

Since this seemed to work on one setup, I'm not sure if this is a suggestion or bug report, but in any case, I feel it's worth mentioning.

The way it is working now, I'm in the position of having to drag hundreds of fuzzy potions to correct my setups.

Example A:

![image](https://github.com/user-attachments/assets/ff327028-9230-439f-ae7e-8949cee9494e)

Example of how most of my setups would probably look if this were true:

![image](https://github.com/user-attachments/assets/be92f839-23b3-4120-8dfa-70ed65a8b040)

**Comment by dillydill123** [comment]
Fuzzy items should be moved to the bottom of the setup already. Core Bank Tags provides the ability to add variation items but adds them in the first available spot. I knew this would not be ideal so I add the items to the layout manually. The fuzzy items are supposed to be added at the bottom. You can see this if you do an auto layout. I'm not sure this works with potion storage though.

An ideal solution would be that bank tags allows for adding items that are in the bank and in the tag but not in the layout to the bottom of the layout. This still would have the problem of custom fuzzy mappings not working.

**Comment by TekkadanPlays** [comment]
Okay, yes, that makes sense. I realized that at the same time that I discovered the 'keep' option bug, and then neglected the auto-layout feature.

However, it seems there is still something quirky about it. I am now going through my presets and enabling the auto-layout presets, and it seems that fuzzy potions are acting a bit strange.

Let's take this old Castillo setup for example:

![image](https://github.com/user-attachments/assets/2a50266a-c677-4630-ae35-4e042296a276)

I will enable fuzzy potions on the 3 types of potions that are used in the setup

![image](https://github.com/user-attachments/assets/9013c7d4-7e05-4be9-9107-28c5956d2cf0)

It seems that Stamina potions and Saradomin Brews are the issue with this particular setup.

And, trying with another setup, it seems those two same potions are affected. So I will finish this off with "sara brews and stamina potions" seem to be working incorrectly for fuzzy. Will keep testing.

**Comment by dillydill123** [comment]
Ah, I see the problem. It's because they are placeholders.

**Comment by TekkadanPlays** [comment]
Perhaps a toggle option in settings could offer us the ability to create a secondary new line for spacing? Would be a lot more visually pleasing in a lot of cases, I think.

**Comment by TekkadanPlays** [comment]
![image](https://github.com/user-attachments/assets/9b7e70b2-9e29-4c08-9881-5c515014f60e)

Another example of the placeholder issue affecting degradable gear

**Comment by dillydill123** [comment]
The placeholder issue will be fixed with the next RuneLite release, since Adam made some changes to bank tags that I'm waiting on.

I think what you are seeing is also a different issue. I explained a little bit in another issue, but fuzzy took a little bit of a hit when it comes to the new layout system. Bank Tags is trying to add items that are in the bank but not in the layout as well, so I try to intercept those and add items at the bottom of the layout. This is not perfect because degradable items and charged jewelry have a specific placeholder that bank tags is trying to add after. I spent some time trying to figure out how to get it to play nice with Bank Tags but failed. I wanted to get the release out and figured I could look into it again so I stopped there.

My recommendation for now is to just leave the extra items in your layout but move them out of the way.

**Comment by dillydill123** [comment]
The placeholders should now be going to the bottom, but the fuzzy and degradable items are still somewhat of an issue. Needs more investigation by me.

**Comment by dillydill123** [comment]
Fuzzy items have been changed with 1.21.0 to behave more like they did in bank tag layouts. Please create a new issue if there are any new problems.
